setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(name='collect_social',
+      packages=['collect_social'],
+      zip_safe=False,
+      license='MIT',
+      )
```

Comment:
I like the idea here. We want to move towards being a packaged module available on pip but I am not sure we're ready for that yet. Since I am not too familiar with the pro/con here, assuming we are not distributing as a package (yet) are there benefits to having setup.py prior to releasing to pip?